snap/README.md

```diff
@@ -110,3 +130,41 @@ $ sudo sed -i -e 's@/vault/config/assets/resp-init.json@/var/snap/edgex-app-serv
 ```
 
 **Note** -- these configuration changes need to be made *before* the service is started for the first time. Otherwise, the recommended approach is to stop the service, delete the existing app-service-configurable configuration in Consul's kv store, and then proceed.
+
+## Service Environment Configuration Overrides
+**Note** - all of the configuration options below must be specified with the prefix: 'env.'
+
+```
+[Service]
+service.boot-timeout            // Service.BootTimeout
+service.health-check-interval   // Service.HealthCheckInterval
+service.host                    // Service.Host
+service.server-bind-addr        // Service.ServerBindAddr
+service.port                    // Service.Port
+service.protocol                // Service.Protocol
+service.max-result-count        // Service.MaxResultCount
+service.max-request-size        // Service.MaxRequestSize
+service.startup-msg             // Service.StartupMsg
+service.request-timeout         // Service.RequestTimeout
+
+[Clients.core-data]
+clients.core-data.host                 // Clients.core-data.Host
+clients.core-data.port                 // Clients.core-data.Port
+clients.core-data.protocol             // Clients.core-data.Protocol
```

Comment:
Note with V2 we now have the possibility of additional Metadata, Command and Notification clients. Not require unless using the related Client interfaces. Also, Metadata has become the require Client for version check to work since Core Data is now optional.
I will be updating ASC profiles to have to appropriate clients.
Here is example from the template config TOML: https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/app-service-template/res/configuration.toml#L73-L92